modules/apigee_edge_teams/src/Entity/Team.php

```diff
@@ -65,6 +65,7 @@
  *   },
  *   entity_keys = {
  *     "id" = "name",
+ *     "label" = "displayName",
```

Comment:
What about other entity types? Is this information also missing from those?

Reply:
@arlina-espinoza the PR has been merged, does it mean that other entity types do not need this change?

Reply:
Also, if entity keys are changing, should not we add an update hook that updates the active entity definition on existing sites? 🤔

Reply:
@arlina-espinoza ?